Tests/StackClusterTests.swift

```diff
@@ -161,7 +163,7 @@ class StackClusterTests: QuickSpec {
                     it("should be able to scroll horizontally") {
                         let vc = nestedStack()
                         vc?.loadViewIfNeeded()
-                        
+                        vc?.view.layoutIfNeeded()
```

Comment:
I think the reason was that nothing is triggering a layout action (because it's not embeded somewhere) - therefore the layoutIfNeeded is needed here, right @AndreasThenn ?